packages/block-editor/src/hooks/use-typography-props.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { kebabCase } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { getInlineStyles } from './style';
+
+// This utility is intended to assist where the serialization of the typography
+// block support is being skipped for a block but the spacing related CSS
```

Comment:
```suggestion
// block support is being skipped for a block but the typography related CSS
```

Missed updating this on the copy/paste.